docs/snippets/common/button-story-hide-nocontrols-warning.js.mdx

```diff
@@ -0,0 +1,7 @@
+```js
+// Button.stories.js
+
+Basic.parameters = {
```

Comment:
I guess we should be consistent here -- in the other cases we include the `Template.bind({});` also.

Reply:
Fixed with the latest commit